use-cases.md

```diff
@@ -158,7 +158,7 @@ When an application acts as an RPC client, it is expected to start a new tracing
             future.add_done_callback(on_done)
             return future
         except Exception e:
-            span.log_event_with_payload('exception', e)
+            span.log('exception', e)
```

Comment:
how would it look in Go, where there's no method overloading or default param values?